src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -20,7 +20,7 @@
       text: az login
     - name: Log in with user name and password. This doesn't work with Microsoft accounts or accounts that have two-factor authentication enabled. Use -p=secret if the first character of the password is '-'.
       text: az login -u johndoe@contoso.com -p VerySecret
-    - name: Log in with a service principal using client secret. Use -p=secret if the first character of the password is '-'.
+    - name: Log in with a service principal using client secret. Use -p=secret if the first character of the password is '-'. For some special characters, including double quotes in the password string is required; in PowerShell this would be -p ``"`"abc[?>def`""``.
```

Comment:
``` ``"`"abc[?>def`""`` ``` is not valid PowerShell syntax:

```
> echo ``"`"abc[?>def`""``
`"abc[?>def"`
```

Actually, PowerShell quoting issue doesn't only happen here for `az login --password`. It happens everywhere. We have  detailed documentation for this topic: https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md